docker/prometheus/prometheus.yml

```diff
@@ -3,15 +3,10 @@ global:
   evaluation_interval: 2s
 
 scrape_configs:
```

Comment:
as we now use the push model, do we need need this scrape config?

Reply:
Prometheus still needs to scrape metrics from Opentelemetry and itself. Exporter only pushes metrics it receives from Receivers (from Agent of Java services )